docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md

```diff
@@ -0,0 +1,63 @@
+# get_raw_transaction
+The  `get_raw_transaction`  method returns hexadecimal encoded signed transactions.
+
+## Arguments
+|Structure|Type|Description|
+|:-------:|:---:|:--------:|
```

Comment:
Not sure if it matters for vuepress, but normally we don't have the `:` for table formating, and try to align the `|` so it's a bit easier to read. eg - 

```


|Param    | Type   | Description           |
|---------|--------|-----------------------|
|field    | string | detail about field    |

```